.travis.yml

```diff
@@ -8,45 +10,53 @@ env:
     - MINIKUBE_HOME=$HOME
     - CHANGE_MINIKUBE_NONE_USER=true
     - KUBECONFIG=$HOME/.kube/config
+
 services:
   - docker
+
 language: go
+
 go:
   - 1.12.5
 
 addons:
   apt:
     update: true
 
+jobs:
+  include:
+    - os: linux
+      arch: amd64
+    - os: linux
+      arch: arm64
+
 before_install:
   - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
   - sudo apt-get update -qq
   - sudo apt-get install --yes -qq gcc
   - sudo apt-get install --yes -qq libudev-dev
+
 before_script:
-  # Download kubectl, which is a requirement for using minikube.
-  - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
-  # Download minikube.
-  - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.35.0/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
-  - mkdir -p $HOME/.kube $HOME/.minikube
-  - touch $KUBECONFIG
-  - sudo minikube start --vm-driver=none --kubernetes-version=v1.13.0
-  - "sudo chown -R travis: /home/travis/.minikube/"
-  # Wait for Kubernetes to be up and ready.
-  - JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until kubectl get nodes -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1; done
-  # Download and initialize helm.
-  #- ./openebs/ci/ubuntu-compile-nsenter.sh && sudo cp .tmp/util-linux-2.30.2/nsenter /usr/bin
-  #- curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
-  #- chmod 700 get_helm.sh
-  #- ./get_helm.sh
-  #- helm init
+  # Download kubectl and setup minkube for running integration tests
+  # Wait for minikube to be up and ready.
+  #    curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.35.0/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/ ;
+  #    mkdir -p $HOME/.kube $HOME/.minikube ;
+  #    touch $KUBECONFIG ;
+  #    sudo minikube start --vm-driver=none --kubernetes-version=v1.13.0 ;
+  #    "sudo chown -R travis: /home/travis/.minikube/" ;
+  #    JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until kubectl get nodes -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1; done ;
+  - if [ "$TRAVIS_CPU_ARCH" == "amd64" ]; then
```

Comment:
this check should go up , we have to install minikube

Reply:
there was a parsing error with one of the below sudo chown commands. You caught this while the code was still in progress. Please check now.

Reply:
ok, got it